### PR TITLE
Service stop should not be called

### PR DESCRIPTION
**Pull request by denysoblohin-okta**
Changes:
- Removes `oktaAuth.stop()` from `<Security>`
- Removes `oktaAuth.start()` from  `<LoginCallback>` after `handleLoginRedirect ` - not needed
- Fixed `useEffect` hook in `<Security>` - don't watch for `restoreOriginalUri ` prop to perform side-effects (start, subscribe, unsubscribe) on oktaAuth
- Updated `okta-auth-js` to latest

Internal ref: [OKTA-492710](https://oktainc.atlassian.net/browse/OKTA-492710) (including deps OKTA-482712 OKTA-482710)

### CODE REVIEW

File: test/jest/loginCallback.test.tsx [hunk 1]
```diff
@@ -70,6 +70,8 @@ describe('<LoginCallback />', () => {
       </Security>
     );
     expect(oktaAuth.handleLoginRedirect).toHaveBeenCalledTimes(1);
+    await Promise.resolve();
```
Comment (jaredperreault-okta) [inline test/jest/loginCallback.test.tsx:73]: What is the purpose of this `Promise.resolve`?

Comment (denysoblohin-okta) [reply test/jest/loginCallback.test.tsx:73]: To wait for `handleLoginRedirect` which is a promise

Comment (jaredperreault-okta) [reply test/jest/loginCallback.test.tsx:73]: You can put the await in the expect call
```javascript
expect(await oktaAuth.handleLoginRedirect)
```
or
```javascript
await expect(oktaAuth.handleLoginRedirect).resolves.toHaveBeenCalled()
```

**Review by jaredperreault-okta — APPROVED** [review]

File: test/jest/loginCallback.test.tsx (hunk 1 above)
Comment (denysoblohin-okta) [reply test/jest/loginCallback.test.tsx:73]:
```js
    await act(async () => {
      mount(
        <Security {...mockProps}>
          <LoginCallback />
        </Security>
      );
    });
```
this works without need of `await Promise.resolve();`

**Pull request closed without merging**